specs/Deployment/BuildInformation.md

```diff
@@ -0,0 +1,226 @@
+- [1. Background](#1-background)
+- [2. Examples](#2-examples)
+  - [2.1. Examples - C++](#21-examples---c)
+  - [2.2. Examples - C](#22-examples---c)
+- [3. API Details](#3-api-details)
+  - [3.1. WindowsAppSDK-VersionInfo.h](#31-windowsappsdk-versioninfoh)
+  - [3.2. WindowsAppSDK-VersionInfo.cs](#32-windowsappsdk-versioninfocs)
+  - [3.3. WindowsAppSDK-VersionInfo.json](#33-windowsappsdk-versioninfojson)
+  - [3.4. WindowsAppSDK-VersionInfo.xml](#34-windowsappsdk-versioninfoxml)
+
+# 1. Background
+
+Windows App SDK supports deployment as a set of MSIX packages for use at runtime, but some
+operations and APIs require build information about the release. This spec defines public symbols
+provided so developers can easily use Windows App SDK APIs at runtime referencing the version of
+Windows App SDK they use at build-time.
+
+For example,  `MddBootstrapInitialize(majorMinorRelease, versionTag, minVersion)` requires...
```

Comment:
Assuming this is code compiled into the client, can we provide default values for these arguments so that when most developers want to just "use the version I'm building with", they can provide no explicit arguments?

Reply:
Not today

Baking those constants into APIs is problematic due to build-chain chicken/egg ordering issues). I have some ideas how we might be able to tackle that in the future (or at least get closer) but it's not a simple or singular answer so not in this pass.